### PR TITLE
feat(documents): [SBS-3442, SBS-3648] Endpoints for documents

### PR DESCRIPTION
**Pull request by andersfylling**
Implements the endpoints found in the playground for /documents: 
 - https://docs.cognite.com/api/playground/#tag/Documents
 - https://cognitedata.atlassian.net/browse/SBS-3442
 - https://cognitedata.atlassian.net/browse/SBS-3648

The document preview endpoint is not included and will be in a different PR.

### CODE REVIEW

**Comment by andersfylling** [comment]
force pushed due to complaints about early commits (I was not following commit message constraints in the beginning).

**Comment by andersfylling** [comment]
I don't know why other projects, like wells and survey, gets reduced coverage here. Anything I can do, related to the new code I'm introducing?

File: packages/beta/src/api/documents/feedbackApi.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  public accept = (ids: number[]) => {
+    return this.postAcceptRejectFeedback(ids, 'accept');
+  };
+
+  public reject = (ids: number[]) => {
+    return this.postAcceptRejectFeedback(ids, 'reject');
+  };
+
+  private async postAcceptRejectFeedback(
+    ids: number[],
```
Comment (polomani) [inline packages/beta/src/api/documents/feedbackApi.ts]:
```suggestion
    ids: IdEither[],
```

File: packages/beta/src/api/documents/feedbackApi.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    endpoint: acceptRejectEndpoint
+  ) {
+    const wrappedIds = ids.map(id => {
+      return <DocumentFeedbackId>{ id: id };
```
Comment (polomani) [inline packages/beta/src/api/documents/feedbackApi.ts]: We don't perform this kind of wrapping in other APIs.
Is there a specific reason for it here?

Comment (andersfylling) [reply packages/beta/src/api/documents/feedbackApi.ts]: It just felt a bit weird having to specify a list of objects that only has a single attribute. So this was only for convenience.

Comment (andersfylling) [reply packages/beta/src/api/documents/feedbackApi.ts]: Updated the code

**Comment by andersfylling** [comment]
blocked by https://github.com/cognitedata/cognite-sdk-js/pull/557

**Comment by andersfylling** [comment]
Seems the failure is related to assets.

**Review by vegardok — APPROVED** [review]

**Pull request merged**